h/static/styles/partials-v2/_search.scss

```diff
@@ -117,18 +114,20 @@
 }
 
 .search-result-bucket__annotation-cards {
-  flex-grow: 1;
```

Comment:
You're right (god this is tricky), previously I found that on _some_ production data the annotation card would grow _too_ big (beyond its `500px`) and squash the stats, but now (after re-adding this `flex-grow: 1`) I don't seem to be able to reproduce that anymore (maybe it's because we have a `max-width: 500px` now). Anyway, I'll remove this deletion